lib/std/start.zig

```diff
@@ -16,7 +16,7 @@ const native_os = builtin.os.tag;
 
 var argc_argv_ptr: [*]usize = undefined;
 
-const start_sym_name = if (native_arch.isMIPS()) "__start" else "_start";
+const start_sym_name = if (native_arch.isMIPS()) "__start" else if (builtin.is_wasi_reactor_exec_model) "_initialize" else "_start";
```

Comment:
Hmm, here's a thought. Maybe we should augment the builtins and put the exec model enum there much like this is the case with `GlobalLinkage`, etc.?

Reply:
Thanks! I believe https://github.com/ziglang/zig/pull/9178/commits/6769659dedde8b53999fda15b0125639cc451c23 addressed this comment.